gluten-core/src/main/scala/io/glutenproject/expression/ExpressionConverter.scala

```diff
@@ -83,6 +83,9 @@ object ExpressionConverter extends SQLConfHelper with Logging {
       udf: ScalaUDF,
       attributeSeq: Seq[Attribute],
       expressionsMap: Map[Class[_], String]): ExpressionTransformer = {
+    if (!udf.udfName.isDefined) {
+      throw new GlutenNotSupportException("UDF name is not found!")
```

Comment:
This is to fix NoSuchElementException that can be thrown in some Spark UTs, where scala UDF is used with no name provided.